edge-util/src/Microsoft.Azure.Devices.Edge.Util/edged/WorkloadClientVersioned.cs

```diff
@@ -51,6 +51,21 @@ abstract class WorkloadClientVersioned
 
         public abstract Task<string> SignAsync(string keyId, string algorithm, string data);
 
+        public virtual Task<string> GetSecretAsync(string secretId) => Task.FromResult(string.Empty);
+
+        public virtual Task SetSecretAsync(string secretId, string secretValue) => Task.CompletedTask;
+
+        public virtual Task DeleteSecretAsync(string secretId) => Task.CompletedTask;
+
+        protected Task Execute(Func<Task> func, string operation) =>
```

Comment:
where is execute used?

Reply:
https://github.com/Azure/iotedge/blob/6a27e91baf3d28989ee972fa2fcc5706c8f82266/edge-util/src/Microsoft.Azure.Devices.Edge.Util/edged/version_2020_07_22/WorkloadClient.cs#L110-L126